docs/articles_en/openvino-workflow/model-optimization-guide/compressing-models-during-training/quantization-aware-training.rst

```diff
@@ -16,83 +16,45 @@ Using NNCF QAT
 ####################
 
 Here, we provide the steps that are required to integrate QAT from NNCF into the training script written with
-PyTorch or TensorFlow 2:
-
-.. note::
-   Currently, NNCF for TensorFlow 2 supports optimization of the models created using Keras
-   `Sequential API <https://www.tensorflow.org/guide/keras/sequential_model>`__ or
-   `Functional API <https://www.tensorflow.org/guide/keras/functional>`__.
+PyTorch:
 
 1. Import NNCF API
 ++++++++++++++++++++
 
 In this step, you add NNCF-related imports in the beginning of the training script:
 
-.. tab-set::
-
-   .. tab-item:: PyTorch
-      :sync: pytorch
+.. tab:: PyTorch
 
-      .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_torch.py
-         :language: python
-         :fragment: [imports]
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_torch.py
+      :language: python
+      :fragment: [imports]
 
-   .. tab-item:: TensorFlow 2
-      :sync: tensorflow-2
 
-      .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
-         :language: python
-         :fragment: [imports]
-
-2. Create NNCF configuration
```

Comment:
why is documentation removed for TensorFlow?

Reply:
This is WIP docs update, we are going to mark `create_compressed_model` quantization method as deprecated and suggest to use `nncf.quantize` method. The way we will reflect this in docs is not discussed and approved yet.

Currently I've removed old method and have replaced it with new one to start the discussion with my team

Reply:
Okay, hope to have documentation for TensorFlow using new method as well.